### PR TITLE
shown mnemonic phrase to wallet create page

### LINKED ISSUE #958: Display Mnemonic Phrase when creating a new wallet
image

### PR DESCRIPTION
**Pull request by gyanpgupta**
/kind ui-change
Fixes #958

### CODE REVIEW

File: src/shared/wallet/new-wallet.tsx [hunk 1]
```diff
@@ -76,13 +127,13 @@ class NewWallet extends React.Component<Props, State> {
             <Input.Password
               className="form-input"
               placeholder={t("wallet.account.addressPlaceHolder")}
-              addonAfter={copyButton}
+              addonAfter={copyPriKeyButton}
               value={account.privateKey}
               readOnly={true}
             />
           </Form.Item>
+          {this.saveMnemonic}
```
Comment (nmhung1210) [inline src/shared/wallet/new-wallet.tsx]: `{this.saveMnemonic()}`

File: src/shared/wallet/new-wallet.tsx [hunk 2]
```diff
@@ -20,33 +23,81 @@ export interface Props extends DispatchProp {
...
+    this.setState({ copiedMnemonic: true });
   };
 
   public setAccount = () => {
     this.props.dispatch(setAccount(this.state.account));
   };
 
-  public render(): JSX.Element {
-    const { account, copied } = this.state;
+  public saveMnemonic = () => {
```
Comment (nmhung1210) [inline src/shared/wallet/new-wallet.tsx]: Pls. use prefix `render` for this.

File: translations/en.yaml [hunk 3]
```diff
@@ -286,6 +286,8 @@ block.show_more: 'Show More'
 block.show_less: 'Show Less'
 new-wallet.download: 'Download Keystore File (UTC / JSON)'
 wallet.cannot_load_solidity_version: 'Cannot load solidity version'
+new-wallet.save: 'Save your'
```
Comment (puncsky) [inline translations/en.yaml]: do not split the sentence.

https://onefx.js.org/contributing.html#avoid-missing-i18n-rtl-issues

Comment (gyanpgupta) [reply translations/en.yaml]: @puncsky  it isn't possible without using dangreouslySetInnerHtml which is giving me error like invalid call to dangreouslySetInnerHtml that's why I have to do it that way.

File: src/shared/wallet/new-wallet.tsx [hunk 4]
```diff
@@ -20,33 +23,81 @@ export interface Props extends DispatchProp {
...
+    this.setState({ copiedMnemonic: true });
   };
 
   public setAccount = () => {
     this.props.dispatch(setAccount(this.state.account));
   };
 
-  public render(): JSX.Element {
-    const { account, copied } = this.state;
+  public saveMnemonic(): JSX.Element {
```
Comment (nmhung1210) [inline src/shared/wallet/new-wallet.tsx]: Pls. help rename function to`renderSaveMnemonic` to avoid confusion.

File: translations/en.yaml (hunk 3 above)
Comment (nmhung1210) [reply translations/en.yaml]: @gyanpgupta pls. ref `new-wallet.save_keystore`. Some languages have differences in the order of words (left to right, right to left).

**Pull request merged**